keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resolvers/NetworkResolver.kt

```diff
@@ -44,32 +38,12 @@ abstract class NetworkResolver<T : Locatable<*>>(
     )
   }
 
-  protected fun SimpleLocations.withResolvedNetwork(): SimpleLocations =
-    copy(vpc = vpc ?: DEFAULT_VPC_NAME)
-
   companion object {
     const val DEFAULT_VPC_NAME = "vpc0"
     const val DEFAULT_SUBNET_PURPOSE = "internal (%s)"
   }
 }
 
-@Component
-class SecurityGroupNetworkResolver(cloudDriverCache: CloudDriverCache) : NetworkResolver<SecurityGroupSpec>(cloudDriverCache) {
-  override val apiVersion: ApiVersion = SPINNAKER_EC2_API_V1
-  override val supportedKind: String = "security-group"
-
-  override fun invoke(resource: Resource<SecurityGroupSpec>): Resource<SecurityGroupSpec> =
-    resource.run {
-      copy(
-        spec = spec.run {
-          copy(
-            locations = locations.withResolvedNetwork()
-          )
-        }
-      )
-    }
-}
```

Comment:
Don't need a `NetworkResolver` implementation for `SecurityGroup` as the only thing it did was default the `vpc`.